src/Tools/dotnet-trace/Extensions.cs

```diff
@@ -20,6 +21,35 @@ public static List<Provider> ToProviders(string providers)
                 new List<Provider>() : providers.Split(',').Select(ToProvider).ToList();
         }
 
+        private static EventLevel GetEventLevel(string token)
+        {
+            if (Int32.TryParse(token, out int level) && level >= 0)
+            {
+                return level > (int)EventLevel.Verbose ? EventLevel.Verbose : (EventLevel)level;
+            }
+
+            else
+            {
+                switch (token)
+                {
+                    case "Critical":
```

Comment:
We should normalize the strings for capitalization so that `CrItIcAl` works just as well as `Critical`.

Reply:
Agreed with John here (Could have a HashSet with ignore casing comparer, and check if input in collection).